tdvt/tdvt/config_gen/gentests.py

```diff
@@ -28,6 +28,13 @@ def get_logical_config_templates(ds_registry):
 def get_logical_config_template(ds_registry, config_name):
     return get_logical_config_templates(ds_registry)[config_name]
 
+def check_logical_config_boolean_attr_value(attrs, key):
+    if key not in attrs:
+        return False
+    if key in attrs and attrs[key] == 'False':
```

Comment:
```suggestion
    if attrs[key] == 'False':
```
With previous statement we know key will be in attrs.

I missed this in the first round, apologies.  Another idea to help us out going forward.  Add another `if attrs[key] == 'True':` that returns True explicitly.  Then in the fallback case where 'True' is returned for backwards compatibility, we could log something to the console to warn users 'True' and 'False' are the expected values.

Reply:
sounds good, fixed.